packages/vxrn/src/exports/prebuild.ts

```diff
@@ -3,60 +3,183 @@ import path from 'node:path'
 import FSExtra from 'fs-extra'
 import { fillOptions } from '../utils/getOptionsFilled'
 import { applyBuiltInPatches } from '../utils/patches'
+/*
+This code block is partially copied from meta owned repos.
+Copyright (c) Facebook, Inc. and its affiliates.
+*/
+const generateForPlatform = async (root: string, platform: 'ios' | 'android', appConfig: any) => {
+  const dest = path.resolve(platform)
+  const appName = appConfig.name
+  const displayName = appConfig.displayName
+  const require = module.createRequire(root)
+  const importPath = require.resolve('@react-native-community/cli/build/tools/walk.js', {
+    paths: [root],
+  })
+  const src = path.join(
+    path.dirname(
+      require.resolve('@react-native-community/template/template/package.json', {
```

Comment:
do we want this in our package.json?

Reply:
we can make it a dependency

Reply:
both of the solutions are not nice one just gives more flexibility in versioning

Reply:
in a later pr sure